platform-hub-web/src/app/layout/shell.html

```diff
@@ -124,8 +124,9 @@ <h1 flex>{{$ctrl.AppSettings.getAppTitle()}}</h1>
             <md-sidemenu-button
               ng-repeat="item in $ctrl.adminNavStates"
               ui-sref="{{item.state}}"
+              ng-if="!_.has(item, 'featureFlag') || (_.has(item, 'featureFlag') && $ctrl.FeatureFlags.isEnabled(item.featureFlag))"
```

Comment:
Can remove the `_.has(item, 'featureFlag')` bit as it'll be true by the time that part of the expression is evaluated.